src/main/java/org/alfresco/rest/core/RestWrapper.java

```diff
@@ -752,14 +749,14 @@ private boolean checkForJsonStatusException(Response returnedResponse) throws Em
             lastException = "";
 
         Object error = returnedResponse.jsonPath().get("status");
-        if (error != null)
-        {
+
+        if(error == null || (error instanceof List && ((List<?>) error).size() == 0)) {
+            return false;
+        }else{
```

Comment:
Same comments as for checkForJsonError.

Reply:
Here error is an object; not a String

Reply:
I don't understand this response. The code looks identical in the two methods.